src/Traits/EwalletTrait.php

```diff
@@ -3,7 +3,6 @@
 
 namespace EoneoPay\PhpSdk\Traits;
 
-use Symfony\Component\Serializer\Annotation\Groups;
 use Symfony\Component\Validator\Constraints as Assert;
 
 trait EwalletTrait
```

Comment:
is the trait missing @Group for serialisation on purpose? As per the readme, does missing @Groups mean these properties won't be sent at all?

Reply:
alright, got it, because creating ewallet does not need any thing to be sent.